README.md

```diff
@@ -37,7 +35,3 @@ Site built with Ruby on Rails utilizing ActiveRecord on Postgresql. Site is prim
 ### Built With
 
  * Rails, Ruby, ActiveRecord, HTML, CSS, ERB, Postgresql, and Heroku.
```

Comment:
Can we add versions that are used for at least the Ruby and Rails?

Reply:
Agreed I think this is something we can add.